tests/validation/tests/v3_api/resource/cluster.yml

```diff
@@ -1,4 +1,5 @@
 ssh_key_path: .ssh/$AWS_SSH_KEY_NAME
+kubernetes_version: $KUBERNETES_VERSION
```

Comment:
in the case no value is passed in, is it ok to leave the line with no value in the cluster.yml file for RKE? 
Can you try this case?

Reply:
@jiaqiluo If no value is passed in, default value for the RKE cli will be chosen